specs/waku/v2/waku-store.md

```diff
@@ -30,19 +31,18 @@ This specification explains the Waku `store` protocol which enables querying of
 
 **Protocol identifier***: `/vac/waku/store/2.0.0-beta2`
 
-## Security Requirements
+# Security Requirements
```

Comment:
We talked about this before, and it is a bit out of scope, but the most important "feature" by far is Best Effort Message Persistence, and the most important assumption is High Available. These are the two facts I personally care most about when it comes to store protocol. Maybe worth capturing in an issue?

Reply:
Certainly, created the issue https://github.com/vacp2p/specs/issues/281